great_expectations/data_context/data_context/cloud_data_context.py

```diff
@@ -330,16 +374,16 @@ def list_expectation_suite_names(self) -> List[str]:
 
     @property
     def ge_cloud_config(self) -> Optional[GXCloudConfig]:
-        return self._ge_cloud_config
+        return self._cloud_config
 
     @property
     def ge_cloud_mode(self) -> bool:
-        return self._ge_cloud_mode
+        return self._cloud_mode
```

Comment:
again, we should introduce a cloud_mode property (and raise a dep warning on this one)